pkg/apis/infinispan/v1/types_util.go

```diff
@@ -13,6 +14,16 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+type ImageType string
+
+const (
+	// Container image based on JDK
+	ImageTypeJava ImageType = "Java"
```

Comment:
Nitpick, but should we call this "JVM"? Both the normal and native images are written in Java, this just determines the runtime.

Reply:
Yes, good point, @ryanemerson . Will fix.